pipelineutilities/pipelineutilities/search_files.py

```diff
@@ -349,6 +351,24 @@ def json_serial(obj):
     raise TypeError("Type %s not serializable" % type(obj))
 
 
+def _add_more_file_fields(json_record: dict, iiif_image_service_uri: str = None) -> dict:
+    """ Add mimeType (if absent), add mediaServer and mediaResourceId """
+    file_path = json_record.get('filePath')
+    if file_path:
+        file_extension = Path(file_path).suffix
+        if file_extension and '.' in file_extension and file_extension.lower() in ['.tif']:
+            json_record['mimeType'] = json_record.get('mimeType', 'image/tiff')
+            file_path_no_extension = os.path.join(Path(file_path).parent, Path(file_path).stem)
+            json_record['mediaResourceId'] = file_path_no_extension.replace('/', '%2F')
+            if not iiif_image_service_uri:
+                iiif_image_service_uri = json_record.get('iiifImageServiceUri', '')
+            if iiif_image_service_uri:
+                json_record['mediaServer'] = iiif_image_service_uri
+        elif file_extension and '.' in file_extension and file_extension.lower() in ['.pdf']:
```

Comment:
Is the '.' comparison necessary? Could this be shortened to:
if file_extension and file_extension.lower() in ['.pdf']: